e2e/cypress/helpers/index.js

```diff
@@ -1,16 +1,102 @@
-
-
 export const Urls = {
     CASES_URL: '/cases',
     NEW_CASE_URL: '/cases/new',
     NEW_ATTRIBUTE_URL: '/settings/attributes/new',
     NEW_CASETYPE_URL: '/settings/casetypes/new',
     NEW_INDIVIDUAL_URL: '/individuals/new',
-    NEW_PARTYTYPE_URL: '/settings/partytypes/new',
+    NEW_ENTITYTYPE_URL: '/settings/partytypes/new',
```

Comment:
Can we stick to `PARTY` ? I know it says Entity in the UI, but we intend to change it.